src/test/java/ru/ayaz/UserTest.java

```diff
@@ -0,0 +1,54 @@
+package ru.ayaz;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by ayaz on 11/5/16.
+ */
+public class UserTest {
+
+    private User user;
+
+    @Before
+    public final void before(){ user = new User(); }
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Test
+    public void passingNull_setName() throws Exception {
```

Comment:
I wouldn't recommend to put the whole contents of the test in its description (method name that is) - test name should convey the general meaning of the test. By looking at it I should be able to understand which piece of functionality is broken when the test is red. I shouldn't have to read through the whole test content as the book suggests me to ;)